string[10] Category
int32[10] PositionX
int32[10] PositionY
int32 count 
